content/en/docs/instrumentation/python/getting-started.md

```diff
@@ -1,249 +1,271 @@
-        "span_id": "0xb2fa4c39f5f35e13",
-        "trace_state": "{}"
-    },
-    "kind": "SpanKind.INTERNAL",
-    "parent_id": "0x77e577e6a8813bf4",
-    "start_time": "2020-05-07T14:39:52.906272Z",
-    "end_time": "2020-05-07T14:39:52.906343Z",
-    "status": {
-        "status_code": "OK"
-    },
-    "attributes": {},
-    "events": [],
-    "links": []
-}
-{
-    "name": "bar",
-    "context": {
-        "trace_id": "0xb51058883c02f880111c959f3aa786a2",
-        "span_id": "0x77e577e6a8813bf4",
-        "trace_state": "{}"
-    },
-    "kind": "SpanKind.INTERNAL",
-    "parent_id": "0x3791d950cc5140c5",
-    "start_time": "2020-05-07T14:39:52.906230Z",
-    "end_time": "2020-05-07T14:39:52.906601Z",
-    "status": {
-        "status_code": "OK"
-    },
-    "attributes": {},
-    "events": [],
-    "links": []
-}
-{
-    "name": "foo",
-    "context": {
-        "trace_id": "0xb51058883c02f880111c959f3aa786a2",
-        "span_id": "0x3791d950cc5140c5",
-        "trace_state": "{}"
-    },
-    "kind": "SpanKind.INTERNAL",
-    "parent_id": null,
-    "start_time": "2020-05-07T14:39:52.906157Z",
-    "end_time": "2020-05-07T14:39:52.906743Z",
-    "status": {
-        "status_code": "OK"
-    },
-    "attributes": {},
-    "events": [],
-    "links": []
-}
-```
+app = Flask(__name__)
 
-Each span typically represents a single operation or unit of work. Spans can be
-nested, and have a parent-child relationship with other spans. While a given
-span is active, newly-created spans inherit the active span’s trace ID, options,
-and other attributes of its context. A span without a parent is called the root
-span, and a trace is comprised of one root span and its descendants.
+@app.route("/rolldice")
+def roll_dice():
+    return str(do_roll())
 
-In this example, the OpenTelemetry Python library creates one trace containing
-three spans and prints it to STDOUT.
+def do_roll():
+    return randint(1, 6)
+```
 
-## Configure exporters to emit spans elsewhere
+When run, this will launch an HTTP server with a `/rolldice` route.
 
-The previous example does emit information about all spans, but the output is a
-bit hard to read. In most cases, you can instead *export* this data to an
-application performance monitoring backend to be visualized and queried. It’s
-also common to aggregate span and trace information from multiple services into
-a single database, so that actions requiring multiple services can still all be
-visualized together.
+## Add automatic instrumentation
 
-This concept of aggregating span and trace information is known as distributed
-tracing. One such distributed tracing backend is known as Jaeger. The Jaeger
-project provides an all-in-one Docker container with a UI, database, and
-consumer.
+Automatic instrumentation will generate telemetry data on your behalf. There are
+several options you can take, covered in more detail in [Automatic
+Instrumentation]({{< relref "automatic" >}}). Here we'll use the
```

Comment:
It is on the site, yeah. I think we will be moving away from `relref`s though. I am likely going to go through and update these to "normal" relative links, but it's everywhere in the repo and so that might be done orthogonally